data/recData/notify.js

```diff
@@ -0,0 +1,69 @@
+class Notify {
+  constructor() {
+    self.port.on('data', recs => {
+      this.removeOldBoxes();
+      this.createNewBoxes(recs);
+    });
+  }
+
+  //removes old recommendations, that may be for a different domain
+  removeOldBoxes() {
+    var recDiv = document.getElementById('recs');
+    while(recDiv.firstChild) {
+      recDiv.removeChild(recDiv.firstChild);
+    }
+  }
+
+
+  createNewBoxes(recs) {
+    for(var i = 0; i < recs.length; i++) {
+      var box = this.createNewBox();
+      this.fillInValues(box, recs[i]);
+    }
+  }
+
+
+  //replace default inner-html values
+  fillInValues(div, data) {
+    div.getElementsByClassName('name')[0].innerHTML = data.name;
```

Comment:
I think it would be better to uniformly use `.querySelector()` to select DOM elements, for example, this would be `div.querySelector('.name').innerHTML = data.name`. You can use `.querySelectorAll()` to select a list of elements.